### DISCUSSION TITLE
Breakpoints doesn't seem to work in venus simulator and vs code

### DISCUSSION THREAD

**Discussion by ajtheb** [hm-riscv/vscode-riscv-venus#108] [General]
I am using breakpoints in visual studio code with risc v venus simulator, but it doesn't stop on breakpoints.
And i have set "stopAtBreakpoints": true in launch.json file.
Any solution?

![image](https://github.com/hm-riscv/vscode-riscv-venus/assets/48976954/d8fc0f8f-9a7e-4647-aa14-08c928f1b159)

https://github.com/hm-riscv/vscode-riscv-venus/assets/48976954/edbb3f36-b3ec-4763-ab10-8bcac570361a

**Comment by wallento** [comment]
This is unfortunately a known bug, only under windows, that was introduced with vscode in February. We will try to solve it before October (semester start here). Until then the only solution is to go back to an older version of vscode.

**Comment by neeljoshii** [comment]
You can downgrade the version to 1.69.2 which will allow you to use breakpoints.
